package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fastify",
-  "version": "2.8.0",
+  "version": "2.8.1",
```

Comment:
Do not change the version.

Reply:
Ok, no problem

Reply:
Can you please revert this?

Reply:
Done https://github.com/fastify/fastify/pull/1872/commits/22c8d791b2b15dca3165df1faad7faa4fce2d765